src/Components/Home/Home.css

```diff
@@ -25,6 +25,11 @@ html {
   background-color: white;
   opacity: 1;
   color: #d90932;
+	line-height: 20px;
+}
+.btn:hover {
+	background-color: #d90932;
```

Comment:
Also, change this (`.btn`) style to make it more specific.
Styling directly to a bootstrap class is a bad practice and bad idea overall, this style is interfering with SignUp/SignIn page too.
We have given ID/Classes to parent container for this reason alone, so that when one is styling they can style only the element they want to.

Reply:
`.btn` is a bootstrap class, if you modify it any other button using that class will also get modified.
You have to select this button more **specifically**. So make use of specificity ie use id/class or select it via hierarchy.